### DISCUSSION TITLE
Server Info Available to Client

### DISCUSSION THREAD

**Discussion by Exergist** [jchristn/SuperSimpleTcp#52] [Q&A]
Is there a way for the client to obtain information about the server process when using the local host (127.0.0.1)? 

For example, App1 acts as client and App2 acts as server ==> can App1 determine the process name associated with App2?

Thanks!

**Comment by jchristn** [comment]
Hi @Exergist not through the library, no.  There are OS mechanisms that you can use, though.  For instance: https://stackoverflow.com/questions/48198/how-can-you-find-out-which-process-is-listening-on-a-tcp-or-udp-port-on-windows
